about.md

```diff
@@ -7,9 +7,7 @@ permalink: /about/
 This is the base Jekyll theme. You can find out more info about customizing your Jekyll theme, as well as basic Jekyll usage documentation at [jekyllrb.com](http://jekyllrb.com/)
 
 You can find the source code for the Jekyll new theme at:
-{% include icon-github.html username="jekyll" %} /
-[minima](https://github.com/jekyll/minima)
+[github.com/jekyll/minima](https://github.com/jekyll/minima)
 
 You can find the source code for Jekyll at
-{% include icon-github.html username="jekyll" %} /
-[jekyll](https://github.com/jekyll/jekyll)
+[github.com/jekyll/jekyll](https://github.com/jekyll/jekyll)
```

Comment:
It doesn't make a difference, but please replace this with text from Jekyll 3.5's `about.md` [template](https://raw.githubusercontent.com/jekyll/jekyll/v3.5.0/lib/site_template/about.md)